# define the goal
std_msgs/Header header
string[] grippers_names
geometry_msgs/Pose[] gripper_poses
float64[] robot_configuration
bool robot_configuration_valid
---
# define the result
deformable_manipulation_msgs/WorldState world_state
